packages/nc-gui-v2/components/smartsheet-toolbar/SearchData.vue

```diff
@@ -27,9 +16,26 @@ const columns = computed(() =>
 </script>
 
 <template>
-  <a-input v-model:value="localValue" size="small" class="max-w-[250px]" placeholder="Filter query">
+  <a-input
+    v-model:value="search.value"
```

Comment:
v-model should be on `search` not `search.value`

Reply:
In the above case `search` is a `reactive({ value: '' })` variable and `value` is a property in object.